src/cryptography/x509/extensions.py

```diff
@@ -2309,11 +2310,15 @@ def __eq__(self, other: object) -> bool:
         )
 
     def __hash__(self) -> int:
+        if self.profession_oids is None:
+            profession_oids: tuple[ObjectIdentifier | None, ...] = (None,)
+        else:
+            profession_oids = tuple(self.profession_oids)
         return hash(
             (
                 self.naming_authority,
                 *tuple(self.profession_items),
-                *tuple(self.profession_oids),
+                *tuple(profession_oids),
```

Comment:
Instead of doing this, I'd suggest:

```
if self.professional_oids is None:
    profession_oids = None
else:
    profession_oids = tuple(self.profession_oids)
```

and then remove the `*` unpacking

Reply:
It's probably not needed, no. If you want to do taht cleanup, let's do it in a separate PR.